openmetadata-ui/src/main/resources/ui/src/components/ServiceInsights/ServiceInsightsTab.tsx

```diff
@@ -0,0 +1,176 @@
+
+  const fetchChartsData = async () => {
+    try {
+      setIsLoading(true);
+      const currentTimestampInMs = getCurrentMillis();
+      const sevenDaysAgoTimestampInMs = getEpochMillisForPastDays(7);
+
+      const chartsData = await getMultiChartsPreviewByName(
+        SERVICE_INSIGHTS_CHART,
+        {
+          start: sevenDaysAgoTimestampInMs,
+          end: currentTimestampInMs,
+          filter: `{"query":{"bool":{"must":[{"term":{"service.name.keyword":"${nameWithoutQuotes}"}}]}}}`,
+        }
+      );
+
+      const platformInsightsChart = PLATFORM_INSIGHTS_CHART.map((chartType) => {
+        const summaryChartData = chartsData[chartType];
+
+        const data = summaryChartData.results;
+
+        const firstDayValue = data.length > 1 ? data[0]?.count : 0;
+        const lastDayValue = data[data.length - 1]?.count;
+
+        const percentageChange =
+          ((lastDayValue - firstDayValue) /
+            (firstDayValue === 0 ? lastDayValue : firstDayValue)) *
+          100;
+
+        const isIncreased = lastDayValue >= firstDayValue;
+
+        return {
+          chartType,
+          data,
+          isIncreased,
+          percentageChange: isNaN(percentageChange)
+            ? 0
+            : round(Math.abs(percentageChange), 2),
+          currentCount: round(last(summaryChartData.results)?.count ?? 0, 2),
+        };
+      });
+
+      const piiDistributionChart =
+        chartsData[SystemChartType.PIIDistribution].results;
+      const tierDistributionChart =
+        chartsData[SystemChartType.TierDistribution].results;
+
+      setChartsResults({
+        platformInsightsChart,
+        piiDistributionChart,
+        tierDistributionChart,
+      });
+    } catch (error) {
+      showErrorToast(error as AxiosError);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    fetchChartsData();
+  }, []);
+
+  const {
+    PlatformInsightsWidget,
+    PIIDistributionWidget,
+    TierDistributionWidget,
+    MostUsedAssetsWidget,
+    MostExpensiveQueriesWidget,
+    DataQualityWidget,
+    CollateAIWidget,
+  } = serviceUtilClassBase.getInsightsTabWidgets(serviceCategory);
+
+  return (
+    <Row className="service-insights-tab" gutter={[16, 16]}>
+      <Col span={24}>
+        <PlatformInsightsWidget
+          chartsData={chartsResults?.platformInsightsChart ?? []}
+          isLoading={isLoading}
+        />
+      </Col>
+      {!isUndefined(CollateAIWidget) && (
+        <Col span={24}>
+          <CollateAIWidget />
+        </Col>
+      )}
+      {!isUndefined(PIIDistributionWidget) && (
+        <Col span={12}>
+          <PIIDistributionWidget
+            chartsData={chartsResults?.piiDistributionChart ?? []}
+            isLoading={isLoading}
+          />
+        </Col>
+      )}
+      {!isUndefined(TierDistributionWidget) && (
+        <Col span={12}>
+          <TierDistributionWidget
+            chartsData={chartsResults?.tierDistributionChart ?? []}
```

Comment:
Instead, can we loop over chart widgets? I am referring to how we did it for CustomDashbaord